packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -345,12 +345,14 @@ class RenderFractionallySizedBox extends RenderProxyBox {
   }
 }
 
-/// Forces child to layout at a specific aspect ratio.
+/// Attempts to size the child to a specific aspect ratio.
 ///
-/// The width of this render object is the largest width permited by the layout
+/// The render object first tries the largest width permited by the layout
 /// constraints. The height of the render object is determined by applying the
 /// given aspect ratio to the width, expressed as a ratio of width to height.
-/// For example, a 16:9 width:height aspect ratio would have a value of 16.0/9.0.
+/// For example, a 16:9 width:height aspect ratio would have a value of
+/// 16.0/9.0. If the maximum width is infinite, the initial width is determined
+/// by applying the aspect ratio to the maximum height.
 ///
 /// For example, given an aspect ratio of 2.0 and layout constraints that
```

Comment:
Two blocks of text in a row start with "For example". Put a newline before the first one. For the second one, replace "For example" with "Now consider a second example:" or some such.